src/resources/js/tickets-attendees.js

```diff
@@ -1,287 +1,290 @@
-jQuery( document ).ready( function( $ ) {
```

Comment:
Quick note on the changes in this file: it's mostly unchanged, but that may not be immediately apparent from GitHub's diff viewer. What I have done though is switch the structure a little so we can make functions accessible to external code. 

So far, the only function I have actually made public is `count_columns()`, so it can be reused from attendee table code running in Event Tickets Plus.